Cargo.toml

```diff
@@ -46,10 +44,10 @@ rustls-tls = [
 [dependencies]
 anyhow = "1.0"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread", "time"] }
-kube = { version = "0.55", default-features = false }
-k8s-openapi = { version = "0.11", default-features = false, features = ["v1_20"] }
+kube = { version = "0.58", default-features = false }
+k8s-openapi = { version = "0.12", default-features = false, features = ["v1_21"] }
 futures = "0.3"
-krator = { version = "0.3", default-features = false }
+krator = { git = "https://github.com/krator-rs/krator", version = "0.3", default-features = false, branch = "main" }
```

Comment:
Please note that I am working on cutting a new version of krator with the API changes. I will come back and update this, but it should not be merged until I do

Reply:
See: https://github.com/krator-rs/krator/pull/45